src/pages/cart/CartSummary.jsx

```diff
@@ -103,7 +125,12 @@ const CartSummary = () => {
             <span>Total Amount: </span>
             <span className="price-detail-value">Rs. {totalAmt}</span>
           </div>
-          <button className="button primary mt-1 checkout-btn">Checkout</button>
+          <button
+            className="button primary mt-1  radius-0 checkout-btn"
+            onClick={() => checkoutHandler()}
```

Comment:
If you dont have any parameter to pass in the click handler then you dont need to create the arrow function and pass the functions call.
You can do like onClick={checkouHandler} it will work

Reply:
Yess... will make the changes